prac_05/state_names.py

```diff
@@ -14,8 +14,8 @@
 
 state_code = input("Enter short state: ").upper()
 while state_code != "":
-    if state_code in CODE_TO_NAME:
+    try:
         print(state_code, "is", CODE_TO_NAME[state_code])
```

Comment:
Could an f-string be more effective here.  Also will the output be neatly alligned?